test/cluster/test_strong_consistency.py

```
@@ -1760,3 +1760,832 @@ async def _run(ops, *, kind):
+            await manager.server_not_sees_other_server(servers[0].ip_addr, dst_server.ip_addr)
+            await manager.api.exclude_node(servers[0].ip_addr, [dst_host_id])
+
+            try:
+                await move_task
+            except Exception as exc:
+                logger.info("move_tablet failed while rollback converged as expected: %s", exc)
+
+            async def rollback_finished():
+                tablet_info = await get_tablet_info(manager, servers[0], ks, table_name, tablet_token)
+                if tablet_info.stage is None:
+                    return True
```

Ruff (0.16.3)

[warning] Missing explicit `return` at the end of function able to return non-`None` value

Add explicit `return` statement

(RET503)